src/containers/NFT/NFTTabs/styles.scss

```diff
@@ -0,0 +1,75 @@
+@import '../../shared/css/variables';
+@import '../../shared/css/table';
+
+.nft-tabs {
+  margin-top: 60px;
+}
+
+.offers-table {
+  position: relative;
+  min-height: 150px;
+
+  table {
+    .no-info-panel {
```

Comment:
Is this supposed to have different styling than the no results display used on the assets table?

Reply:
The “no information” for offers table is unique afaik